app/javascript/dashboard/modules/search/components/SearchView.vue

```diff
@@ -147,34 +166,54 @@ export default {
       );
     },
   },
+  beforeDestroy() {
+    this.query = '';
+    this.$store.dispatch('conversationSearch/clearSearchResults');
+  },
   methods: {
     search(q) {
       this.query = q;
       this.$track(CONVERSATION_EVENTS.SEARCH_CONVERSATION);
       this.$store.dispatch('conversationSearch/fullSearch', { q });
     },
     onBack() {
-      this.$router.push({ name: 'home' });
+      this.$emit('close');
+    },
+    closeSearch() {
+      this.$emit('close');
     },
   },
 };
 </script>
 
 <style lang="scss" scoped>
-.search-root {
-  max-width: 800px;
+.backdrop {
+  position: fixed;
+  top: 0;
   width: 100%;
-  margin: 0 auto;
   height: 100%;
+  background: var(--white-transparent);
+  z-index: var(--z-index-very-high);
```

Comment:
The backdrop does not apply to the entire screen, only to the main conversation area. Is this intentional?

<img width="600" alt="image" src="https://user-images.githubusercontent.com/18097732/224645215-b153183d-7eda-4d0d-9402-607be52c83bb.png">